lib/src/main/java/uk/gov/di/ipv/cri/address/library/domain/RawSessionRequest.java

```diff
@@ -0,0 +1,27 @@
+package uk.gov.di.ipv.cri.address.library.domain;
+
+import com.fasterxml.jackson.annotation.JsonAlias;
+
+public class RawSessionRequest {
+    @JsonAlias("client_id")
```

Comment:
Are we ever going to re-serialize this?
If so, we should use @JsonProperty instead

Reply:
At present, there is no requirement to re-serialise